### DISCUSSION TITLE
Searx install script - ERROR: ./utils/searx.sh: line 385: user 'searx' missed read permission: /root/searx

### DISCUSSION THREAD

**Discussion by beatquantum** [searx/searx#2598] [Installation issue: Official wiki]
Version: Git clone available on 24 Feb 2021

OUTPUT OF git log -1

commit 5f4a085fc44ed64499d1aa5c2e29d2f861037b5f (HEAD -> master, origin/master, origin/HEAD)
Merge: 94c9320c 93d1da49
Author: Alexandre Flament <alex@al-f.net>
Date:   Tue Feb 23 17:22:37 2021 +0100

    Merge pull request #2595 from dalf/update-wikidata-units

    [mod] update wikidata_units.json and fetch_wikidata_units.py

``` 
**How did you install Searx?** As per https://searx.github.io/searx/admin/installation.html

sudo -H ./utils/searx.sh install all

**What happened?** The scripts halts after showing the following on the cli.
Clone searx sources
-------------------

ls: cannot access '/root/searx': Permission denied
ERROR: ./utils/searx.sh: line 385: user 'searx' missed read permission: /root/searx

**How To Reproduce**
Try running the script on a fresh Ubuntu 20.04 vps.

**Expected behavior**
Script should complete without error.

Other information: I ran the command as root.

**Comment by return42** [comment]
[For the installation procedure, use a sudoer login to run the scripts. If you install from root, take into account that the scripts are creating a searx, a filtron and a morty user. In the installation procedure these new created users do need read access to the clone of searx, which is not the case if you clone into a folder below /root.](https://searx.github.io/searx/admin/installation.html)

**Reply by miguelgargallo** [reply]
Okay, it worked, but I want to learn, why on tmp it works?

**Reply by stephenhodgkiss** [reply]
The documentation states the following: - 

Jump to a folder that is readable by others and start to clone SearXNG, alternatively you can create your own fork and clone from there.
